src/resources/js/admin-image-field.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable es5/no-arrow-functions */
+/* eslint-disable es5/no-es6-methods */
+jQuery( $ => {
```

Comment:
I love the usage of ES6, which we should continue, but we should also use the `obj` structure to expose this script globally.

Example on TEC: https://github.com/the-events-calendar/the-events-calendar/blob/master/src/resources/js/views/events-bar.js

Make sure all the callbacks are functions we can re-use.

All the selectors should be stored on `obj.selectors` so we can easily modify and indentify.